azresources/containers/aks/aks-with-cmk.bicep

```diff
@@ -86,6 +92,18 @@ param akvResourceGroupName string
 @description('Azure Key Vault Name.  Required when useCMK=true.')
 param akvName string
 
+var podCidra = contains(networkPlugin, 'azure') ? null : podCidr
```

Comment:
What does "a" in `podCidra` mean?  Can you write out the variable in full so it's self describing?